paparazzi/src/main/java/app/cash/paparazzi/Environment.kt

```diff
@@ -68,7 +69,8 @@ fun detectEnvironment(): Environment {
     compileSdkVersion = configLines[2].toInt(),
     resourcePackageNames = configLines[5].split(","),
     localResourceDirs = configLines[6].split(",").map { projectDir.resolve(it).toString() },
-    libraryResourceDirs = configLines[7].split(",").map { artifactsCacheDir.resolve(it).toString() }
+    moduleResourceDirs = configLines[7].split(",").filter { it.isNotBlank() }.map { projectDir.resolve(it).toString() },
```

Comment:
```suggestion
    moduleResourceDirs = configLines[7].split(",").filter { it.isNotBlank() }.map { projectDir.resolve(it).toString() },
```

why is the `filter { it.isNotBlank() }` needed?  can we omit?